backends/lakeroad/example.ys

```diff
@@ -4,4 +4,7 @@ module test(input [1:0] a, input b, output o);
 endmodule
 EOF
 
-write_lakeroad
\ No newline at end of file
+# Write output to file instead of stdout
+write_lakeroad file.egg
+# Write output to stdout
+# write_lakeroad
```

Comment:
```suggestion
write_lakeroad
write_lakeroad file.egg
!rm file.egg
```

We might as well test both writing to a file and writing to stdout. Also added code solving my requested change above.